docs/GlutenUsage.md

```diff
@@ -12,11 +12,10 @@ Based on the different environment, there are some parameters can be set via -D
 | backends-velox                                        | Add -Pbackends-velox in maven command to compile the JVM part of Velox backend                                                                                                     | false                                              |
 | backends-clickhouse                                   | Add -Pbackends-clickhouse in maven command to compile the JVM part of ClickHouse backend                                                                                           | false                                              |
-| build_velox                                           | Enable or Disable building the CPP part of Velox backend                                                                                                                           | OFF                                                |
-| velox_home (only valid when build_velox is ON)        | The path to the compiled Velox project. When building Gluten with Velox, if you have an existing Velox, please set it.                                                             | /PATH_TO_GLUTEN/ep/build-velox/build/velox_ep      |
+| build_velox_backend                                           | Enable or Disable building the CPP part of Velox backend                                                                                                                           | OFF                                                |
+| velox_home (only valid when build_velox_backend is ON)        | The path to the compiled Velox project. When building Gluten with Velox, if you have an existing Velox, please set it.                                                             | /PATH_TO_GLUTEN/ep/build-velox/build/velox_ep     |
 | compile_velox(only valid when velox_home is assigned) | recompile exising Velox use custom compile parameters                                                                                                                              | OFF                                                |
-| velox_build_type                                      | The build type Velox was built with from source code. Gluten uses this value to locate the binary path of Velox's binary libraries.                                                | release                                            |
-| debug_build                                           | Whether to generate debug binary library from Gluten's C++ codes.                                                                                                                  | OFF                                                |
+| velox_build_type                                      | The build type Velox was built with from source code. Gluten uses this value to locate the binary path of Velox's binary libraries.                                                | release (option: release/debug/relWithDebInfo)     |
```

Comment:
Let's remove the option and search the 3 release types from velox_home directly. If we didn't find it, print error and quit build.

Reply:
Ok, Will fix it.